src/logging/LoggedCommand.java

```diff
@@ -0,0 +1,29 @@
+package logging;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import static logging.LogManager.isLoggingEnabled;
+
+public class LoggedCommand extends Command {
+
+    private String name = getClass().toString();
+    private String requirements = getRequirements().toString();
+
+    @Override
+    public void initialize() {
+        if (isLoggingEnabled) {
+            System.out.println("[" + System.currentTimeMillis() + "] - (" + requirements + ") " + name);
```

Comment:
line 14 and like 22 are the same, shouldn't it use the same function?

Reply:
is something like this good?

*check newest commit*